Donation-platform/src/main/java/com/google/Charity.java

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.Comparator;
+
+public final class Charity implements Comparator<Charity>{
```

Comment:
is this class setting trending score? Then rename it to more to the functionality of the class for eg: GetTrendingScoreForCharity.java

Reply:
the idea is to have other methods that revolve around the Charity object in this class, but the only one I have needed so far was the setTrendingScore(). I believe that Toni also has some stuff that overlaps with this class so we will address the full required functionality when we're merging.